packages/sdk/src/modules/did/index.ts

```diff
@@ -198,12 +226,21 @@ export class Did extends Base {
       this._getDidId(options.didControllerAddress.toString())
     );
 
-    const { verificationId, verificationMethod } =
-      this._createVerificationMethod(options.didAccountAddress.toString());
-
-    document.addVerificationmethods(verificationMethod);
+    if (options.customDocumentFields?.verifications) {
```

Comment:
Code was changed so that for if a verification, signature, or service field is set in the customDocumentFields it will iterate through all the objects, else set empty if not present.